pkg/docker/hub/tests/init/hello_world.sql

```diff
@@ -0,0 +1 @@
+select version();
```

Comment:
pkg/docker/hub/tests/custom_init.yml mounts pkg/docker/hub/tests/init to /docker-entrypoint-init.d
the entrypoint runs all scripts in this folders, so if you run the custom_init.yml with docker-compose, you will see the version query in the docker logs.